introduction.md

```diff
@@ -3,3 +3,9 @@
 I am Mike. I like the ninja class in Final Fantasy V and Final Fantasy Tactics.
 
 My favorite Code School path is JavaScript.
+
+Deadly skills
+*Git
+*JavaScript
+*Ruby
+*HTML/CSS
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?